lib/membrane_rtc_engine/endpoints/webrtc/connection_prober.ex

```diff
@@ -23,7 +41,7 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.ConnectionProber do
     {:ok, state}
   end
 
-  @impl GenServer
+  @impl true
   def handle_cast({:bandwidth_estimation, estimation}, state) do
     if state.bitrate_timer, do: :timer.cancel(state.bitrate_timer)
 
```

Comment:
One thing that came up to my mind is that using :timer.send_interval in 48 might be dangerous. It will send a message even if we haven't processed the previous one yet. The question is whether 10ms is long enough to iterate over track receivers and send probes. I think that Process.send_after should be safer

Reply:
I think it is fine for the moment and I would leave it like that with a fixme tag. There is actually another, much bigger issue that this change wouldn't fix: :send_padding is asynchronous and we're not incrementing bytes_sent right away, because we're not sure if the padding packet will be sent, so we might ask the track receiver to send two probes when we only needed one. I think the best way to fix it is to ensure that track receiver actually sends all requested padding packets as soon as possible, instead of just discarding the request.

Reply:
also, this might actually not be frequent enough. Because we're discarding paddings inside frame boundaries, we need to get lucky to actually send the padding packet